### PR TITLE
Mz replace hardcoded values with constants

### PR DESCRIPTION
**Pull request by mz3944**

### CODE REVIEW

File: app/models/checklist.rb [hunk 1]
```diff
@@ -1,8 +1,6 @@
 class Checklist < ActiveRecord::Base
   include SearchableModel
-  validates :name,
-    presence: true,
-    length: { maximum: 50 }
+  validates :name, presence: true, length: { maximum: TEXT_MAX_LENGTH }
```
Comment (ZmagoD) [inline app/models/checklist.rb]: put multiple params in separate line

File: app/models/checklist_item.rb [hunk 2]
```diff
@@ -1,7 +1,5 @@
 class ChecklistItem < ActiveRecord::Base
-  validates :text,
-    presence: true,
-    length: { maximum: 1000 }
+  validates :text, presence: true, length: { maximum: TEXT_MAX_LENGTH }
```
Comment (ZmagoD) [inline app/models/checklist_item.rb]: same as before

File: app/models/comment.rb [hunk 3]
```diff
@@ -1,9 +1,7 @@
 class Comment < ActiveRecord::Base
   include SearchableModel
 
-  validates :message,
-    presence: true,
-    length: { maximum: 1000 }
+  validates :message, presence: true, length: { maximum: TEXT_MAX_LENGTH }
```
Comment (ZmagoD) [inline app/models/comment.rb]: here again

File: app/models/experiment.rb [hunk 4]
```diff
@@ -18,10 +18,9 @@ class Experiment < ActiveRecord::Base
                        if: :workflowimg_check
 
   validates :name,
-            presence: true,
-            length: { minimum: 4, maximum: 50 },
+            length: { minimum: NAME_MIN_LENGTH, maximum: NAME_MAX_LENGTH },
```
Comment (ZmagoD) [inline app/models/experiment.rb]: here too 
length: { minimum: 4,
              maximum: 50}

File: app/models/my_module_group.rb [hunk 5]
```diff
@@ -1,9 +1,7 @@
 class MyModuleGroup < ActiveRecord::Base
   include SearchableModel
 
-  validates :name,
-    presence: true,
-    length: { maximum: 50 }
+  validates :name, presence: true, length: { maximum: NAME_MAX_LENGTH }
```
Comment (ZmagoD) [inline app/models/my_module_group.rb]: the same

File: app/models/user.rb [hunk 6]
```diff
@@ -16,11 +16,13 @@ class User < ActiveRecord::Base
...
 
-  validates :full_name, presence: true, length: { maximum: 50 }
-  validates :initials, presence: true, length: { minimum: 1, maximum: 4 }
+  validates :full_name, presence: true, length: { maximum: NAME_MAX_LENGTH }
+  validates :initials,
+    presence: true,
+    length: { maximum: USER_INITIALS_MAX_LENGTH }
   validates_attachment :avatar,
     :content_type => { :content_type => ["image/jpeg", "image/png"] },
-    :size => { :less_than => 200.kilobytes }
```
Comment (ZmagoD) [inline app/models/user.rb]: use new sintax

**Comment by mz3944** [comment]
I think arguments styling should be specified more precisely before changing anything. SO I'll leave it as is.

**Pull request merged**